### ISSUE TITLE
BUG: missing Ni24.D

### ISSUE THREAD

**Issue by CJ-Wright** [xpdAcq/xpdAcq#442]
Something odd is happening in the xpdAcq install. It seems to be missing the site data?
https://travis-ci.org/xpdAcq/xpd-shadow/builds/284836471#L1223

**Comment by chiahaoliu** [comment]
This seems like the same problem we had with xpdsim before. It might have to do with ``package_resourse`` since ``Ni24.D`` does exist in the directory. 

I will try to reproduce the error first.

**Comment by CJ-Wright** [comment]
Thank you very much!

**Comment by chiahaoliu** [comment]
@CJ-Wright can you rebuild again? That PR of including ``tests`` as ``xpdacq`` package resource wasn't pulled in when you tested.

**Comment by CJ-Wright** [comment]
Rebuilding

**Comment by chiahaoliu** [comment]
Okie, confirmed it has to do with the syntax. I will try to fix at the ``acq`` side and let you know.

**Comment by chiahaoliu** [comment]
@CJ-Wright round 2?

**Comment by CJ-Wright** [comment]
Now the excel file seems to be missing
```
E           FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/mc/envs/testenv/lib/python3.5/site-packages/xpdacq/tests/300000_sample.xlsx'
```
https://travis-ci.org/xpdAcq/xpd-shadow/builds/284836471#L1171

**Comment by chiahaoliu** [comment]
Great, at least we identified one part of the problem.

A note for my future reference (if PR #444 is able to fix our problem):

``pkg_resource`` is a bit tricky. Unless the resource files are specified as ``package_data`` in ``setup.py`` (and follows standard syntax from official docs), these files won't be available for other repos. However, if we are referencing the files within the repo, ``pkg_resource`` can access for all files even if they are not specified in ``setup.py``.

Long story short, the keys for this kind of problem:
1. proper syntax in referencing package data (e.g. ``xpsim`` and here)
2. explicit specifications of files in ``setup.py``

**Comment by chiahaoliu** [comment]
I think this issue has been resolved in PR #444. I will close for now. Reopen if needed.
